src/mb_caa_dimensions/style.tsx

```diff
@@ -0,0 +1,8 @@
+import USERSCRIPT_NAME from 'consts:userscript-name';
```

Comment:
That's the constant you renamed into USERSCRIPT_ID in #458, so this has to be updated.

Reply:
It was actually renamed in #458, which isn't merged yet. So I guess we should wait until that is merged, merge `main` into here again, then do the necessary adjustments.